apps/ai-image-generator/backend/src/upload-images.ts

```diff
@@ -0,0 +1,54 @@
+import { PlainClientAPI, UploadProps } from 'contentful-management';
+import { ImageWithStream, ImageWithUpload } from './types';
+
+const UPLOAD_DOMAIN: Record<string, URL> = {
+  us: new URL('https://s3.us-east-1.amazonaws.com/upload-api.contentful.com'),
+  eu: new URL('https://s3.us-east-1.amazonaws.com/upload-api.contentful.com'),
```

Comment:
should this have the eu URL?.... (ah nevermind just noticed the TO-DO to handle EU below)